ui/tutorial-renderer/src/main/res/layout/tutorial_onboarding_ready.xml

```diff
@@ -33,16 +33,18 @@
             app:layout_constraintTop_toTopOf="parent"
             app:srcCompat="@drawable/bkg_tutorial_onboarding_wave_3" />
 
-        <ImageView
+        <com.airbnb.lottie.LottieAnimationView
             android:id="@+id/image"
```

Comment:
I left this as image because if I changed it to ```animation``` I noticed a significant styling difference that I don't believe is desired.

Reply:
yup, the positioning layout references the image view directly.